src/test/scala/cromwell/webservice/CromwellApiServiceSpec.scala

```diff
@@ -344,12 +349,8 @@ class CromwellApiServiceSpec extends FlatSpec with CromwellApiService with Scala
     Post("/workflows/$version", FormData(Seq("wdlSource" -> HelloWorld.wdlSource(), "workflowInputs" -> CromwellApiServiceSpec.MalformedInputsJson))) ~>
       submitRoute ~>
       check {
-        assertResult("Expecting JSON object for workflowInputs and workflowOptions fields") {
-          responseAs[String]
-        }
-        assertResult(StatusCodes.BadRequest) {
-          status
-        }
+        assertResult(StatusCodes.BadRequest) { status }
```

Comment:
Change these if you'd like, but FYI, IntelliJ's default auto formatter will restore the blocks to a separate line.